main.go

```diff
@@ -30,7 +30,8 @@ var (
 	userPeerSubnet                *net.IPNet
 	leaserSyncInterval            time.Duration
 	ipLeaseTime                   = os.Getenv("WGS_IP_LEASE_TIME")
-	leasesFilename                = os.Getenv("WGS_IP_LEASEs_FILENAME")
+	leasesFilename                = os.Getenv("WGS_IP_LEASES_FILENAME")
+	ups                           = os.Getenv("WGS_ADDRESS")
```

Comment:
Do we need to add another global var here? )

Reply:
Let's ignore this for now actually, it's going away with the changes to the server config file format.